trace/tests/unit/ext/django/app/views.py

```diff
@@ -71,6 +74,29 @@ def mysql_trace(request):
         return HttpResponse(msg)
 
 
+def postgresql_trace(request):
+    try:
+        conn = psycopg2.connect(
+            host='localhost',
+            user='postgres',
+            password=PASSWORD,
+            dbname='test')
+        cursor = conn.cursor()
+
+        query = 'SELECT * FROM company'
+        cursor.execute(query)
+
+        result = []
+
+        for item in cursor.fetchall():
+            result.append(item)
+
+        return HttpResponse(str(result))
+
+    except Exception:
+        return "Query failed. Check your env vars for connection settings."
```

Comment:
This should return a non-200 HTTP status code.  Probably 500.

Reply:
Done.